pyrobosim/pyrobosim/data/test_world.yaml

```diff
@@ -22,12 +22,15 @@ robots:
     location: kitchen
     pose: [0, 0, 0]
     # # Rapidly-expanding Random Tree (RRT) planner
-    # path_planner:
-    #   type: rrt
-    #   max_connection_dist: 0.25
-    #   bidirectional: true
-    #   rrt_star: true
-    #   rewire_radius: 1.0
+    path_planner:
+      type: rrt
+      # occupancy_grid:
+      #   resolution: 0.05
+      #   inflation_radius: 0.15
+      max_connection_dist: 0.25
+      bidirectional: true
+      rrt_star: false
```

Comment:
Sorry, another comment -- can you make `rrt_star: true` in this file so the examples work the same as before?